<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDoubleSpinBox" name="doubleSpinBox_diag">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>340</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_answer">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>861</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_new">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_price">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="print_result">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_price">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eco_class">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eco class:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_answer1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>400</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_diag">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Diag:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_year">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1876</number>
    </property>
    <property name="maximum">
     <number>2024</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>360</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_create_new_phone">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>1271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>72</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>CREATE NEW SMARTPHONE</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>true</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>false</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_year">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1325</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
